src/sensors/utils.rs

```diff
@@ -1,23 +1,412 @@
-use docker_sync::container::Container;
-use k8s_sync::Pod;
-use procfs::process::Process;
+#[cfg(target_os = "linux")]
+use procfs::{self, process::Process};
 use regex::Regex;
+#[cfg(feature = "containers")]
 use std::collections::HashMap;
+#[cfg(target_os = "windows")]
+use sysinfo::{
+    get_current_pid, Process, ProcessExt, ProcessStatus, ProcessorExt, System, SystemExt,
+};
+//use std::error::Error;
+use ordered_float::*;
+use std::path::PathBuf;
 use std::time::{Duration, SystemTime};
+#[cfg(all(target_os = "linux", feature = "containers"))]
+use {docker_sync::container::Container, k8s_sync::Pod};
+
+pub struct IStatM {
+    pub size: u64,
+    pub resident: u64,
+    pub shared: u64,
+    pub text: u64,
+    pub lib: u64,
+    pub data: u64,
+    pub dt: u64,
+}
+
+#[derive(Debug, Clone)]
+pub struct IStat {
+    pub pid: i32,
+    pub comm: String,
+    pub state: char,
+    pub ppid: i32,
+    pub pgrp: i32,
+    pub session: i32,
+    pub tty_nr: i32,
+    pub tpgid: i32,
+    pub flags: u32,
+    //pub minflt: u64,
```

Comment:
comments to remove ? Here and below

Reply:
I agree it's not that pretty to have all those comments, but this is kind of a reminder to explore what procfs and sysinfo have to offer.